.flowconfig

```diff
@@ -6,10 +6,11 @@
 ./src/
 
 [libs]
+./node_modules/fusion-core/flow-typed
```

Comment:
Would love to hear thoughts on how this has worked out for you so far.  It feels like a nice solution for Fusion-modules if we maintain this standard, although it does deviate from using something like [flow-typed](https://github.com/flowtype/flow-typed).